Code/Tools/AssetProcessor/native/ui/MainWindow.ui

```diff
@@ -314,7 +314,7 @@
        <number>8</number>
       </property>
       <item>
-       <widget class="AzQtComponents::SegmentBar" name="buttonList">
+       <widget class="AzQtComponents::SegmentBar" name="buttonList" native="true">
```

Comment:
Authors note, the QT designer added this automatically, I think its meant to be this way

Reply:
I'll revert it, and line up those checkboxes while I'm at it